virtManager/xmleditor.py

```diff
@@ -7,13 +7,24 @@
 from virtinst import log
 
 # We can use either gtksourceview3 or gtksourceview4
+have_gtksourceview = True
 try:
     gi.require_version("GtkSource", "4")
     log.debug("Using GtkSource 4")
 except ValueError:  # pragma: no cover
-    gi.require_version("GtkSource", "3.0")
-    log.debug("Using GtkSource 3.0")
-from gi.repository import GtkSource
+    try:
+        gi.require_version("GtkSource", "3.0")
+        log.debug("Using GtkSource 3.0")
+    except:
```

Comment:
Here we should catch only "ValueError" as that is used to indicate if it's not available.